File: Guard/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n = 0;
    int p = 0;
    do
    {
        n = get_int("Ingrese la Altura: ");
    }
    while (n < 1 || n > 8);
    p = n;
    for (int i = 0; i < n; i++)
    {
        for (int j = 1; j < p ; j++)
        {
            printf(" ");
        }
        p--;
        switch (i)
        {
            case 0 :
                printf("#  #\n");
                break;
            case 1 :
                printf("##  ##\n");
                break;
            case 2 :
                printf("###  ###\n");
                break;
            case 3 :
                printf("####  ####\n");
                break;
            case 4 :
                printf("#####  #####\n");
                break;
            case 5 :
                printf("######  ######\n");
                break;
            case 6 :
                printf("#######  #######\n");
                break;
            case 7 :
                printf("########  ########\n");
                break;
        }
    }
}

File: Guard/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

x = 0
while x < 1 or x > 8:
    x = get_int('ingrese la altura: ', )
l = x
x = 1
for k in range(l):
    if l > 0:
        o = l - 1
        for s in range(o):
            print(' ', end='')

    for i in range(x):
        print('#', end='')

    for j in range(1):
        print('  ', end='')

    for p in range(x):
        print('#', end='')

    x += 1
    l -= 1
    print()